CMakeLists.txt

```diff
@@ -72,9 +72,11 @@ set(libraries
 
 find_package(PkgConfig REQUIRED)
 find_package(Eigen3 REQUIRED)
+set(FPHSA_NAME_MISMATCHED 1) # Suppress warnings, see https://cmake.org/cmake/help/v3.17/module/FindPackageHandleStandardArgs.html
```

Comment:
Should we just update names?

Reply:
I changed upper and lower case letters for SuiteSparse library at https://github.com/SteveMacenski/slam_toolbox/pull/494/commits/76cba46c03f3b88534e1ee0ac05627b7ddb41654.